misc/scripts/deploy-tools.sh

```diff
@@ -222,7 +222,7 @@ function cleanvm {
 function cleanupVMPre {
     case $FILE_EXT in
     deb)
-        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null"
+        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null 2>&1"
```

Comment:
> dpkg messages are OK, tar timing ones are not.

I am going to keep changes for turning off harmless "dpkg message" and will remove the proposed change for tar command that I used .. `tar -xm` .. will revert the file and #1556 keep the issue open until we fix the race condition.